assets/src/edit-story/components/fontPicker/fontList/index.js

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+  useCallback,
+  useLayoutEffect,
+} from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useFont } from '../../../app/font';
+import useFocusOut from '../../../utils/useFocusOut';
+import {
+  createFontFilter,
+  isKeywordFilterable,
+  getOptions,
+  addUniqueEntry,
+  getInset,
+} from './utils';
+import { List, Group, GroupLabel, Option, Selected, NoResult } from './styled';
```

Comment:
Yeah, we've discussed this a few times, and this is definitely one way to reduce complexity by simply removing the "noise" of all the CSS. The counterargument is, that it makes it harder to find the definitions when you need them, but that's a minor concern I think. Let's stick with this and if/when we make a decision on a general approach to this later, we can revisit this one.

Reply:
sounds good. yea I agree with the counterargument as well. Fine with sticking with whatever consensus the team comes to